src/index.tsx

```diff
@@ -11,17 +15,57 @@ const LINKING_ERROR =
   '- You rebuilt the app after installing the package\n' +
   '- You are not using Expo Go\n';
 
-type PagecallWebProps = {
-  color: string;
+export type PagecallWebViewProps = {
   uri: string;
-  style: ViewStyle;
+  style?: ViewStyle;
+  ref?: React.Ref<React.ComponentClass<PagecallWebViewProps>>;
+  onMessage?: (message: string) => void;
+  onNativeEvent?: (event: { nativeEvent: { message: string } }) => void;
+};
+
+export type PagecallWebViewRef = {
+  sendMessage: (message: string) => void;
 };
 
 const ComponentName = 'PagecallWebviewView';
 
-export const PagecallWebView =
+const PagecallWebviewView =
   UIManager.getViewManagerConfig(ComponentName) != null
-    ? requireNativeComponent<PagecallWebProps>(ComponentName)
+    ? requireNativeComponent<PagecallWebViewProps>(ComponentName)
```

Comment:
[Minor]
Remove line 23 and apply this

```suggestion
    ? requireNativeComponent<PagecallWebViewProps & {
      onNativeEvent?: (event: { nativeEvent: { message: string } }) => void;
    }>(ComponentName)
```